ramenpos/src/main/java/io/mx51/ramenpos/FormMain.java

```diff
@@ -1194,96 +1416,105 @@ public void windowDeactivated(WindowEvent e) {
      */
```

Comment:
@RaymondK-mx51 since the code starting from here is generated automatically I would suggest skipping them

Reply:
The same is true about all .form files